addons/mail/static/src/xml/systray.xml

```diff
@@ -31,7 +24,7 @@
     <t t-name="mail.chat.ChannelsPreview">
         <t t-foreach="channels" t-as="channel">
             <div t-attf-class="o_mail_channel_preview #{channel.unread_counter ? 'o_channel_unread' : ''}"
-                 t-att-data-channel_id="channel.id">
+                 t-att-data-channel_id="channel.id" t-att-data-message_id="channel.message_id">
```

Comment:
@tivisse  , On click of inbox message(which is loaded in chat window) redirect to that object(task, slae order etc...) using this message_id : https://github.com/odoo/odoo/pull/16739/files#diff-bb8f64175835a0c280e6ef3c680d5679R130